### ISSUE TITLE
Reenable AzureFile tests for Windows on K8s 1.11

### ISSUE THREAD

**Issue by PatrickLang** [Azure/acs-engine#3439]
Once this is cherry picked & released in v1.11
https://github.com/kubernetes/kubernetes/pull/65893

Remove the skip:
https://github.com/Azure/acs-engine/blob/677ae560214b73112acdb3aece8fe8f3631b6696/test/e2e/kubernetes/kubernetes_test.go#L816

**Comment by PatrickLang** [comment]
Fix merged into v1.11, just waiting on a new release
